.vscode/settings.json

```diff
@@ -9,11 +9,22 @@
     "statusBar.background": "#b85833",
     "statusBarItem.hoverBackground": "#ce7350",
     "statusBar.foreground": "#e7e7e7",
-    "statusBar.border": "#b85833"
+    "activityBar.activeBackground": "#ce7350",
+    "activityBar.activeBorder": "#82dc9b",
+    "activityBar.background": "#ce7350",
+    "activityBar.foreground": "#15202b",
+    "activityBar.inactiveForeground": "#15202b99",
+    "activityBarBadge.background": "#82dc9b",
+    "activityBarBadge.foreground": "#15202b",
+    "titleBar.activeBackground": "#b85833",
+    "titleBar.activeForeground": "#e7e7e7",
+    "titleBar.inactiveBackground": "#b8583399",
+    "titleBar.inactiveForeground": "#e7e7e799"
```

Comment:
Was this from peacock?

Reply:
I need to find out how to disable this behavior. Yes, it is from peacock

Reply:
I think if it detects that you're setting the status bar colour it takes over and says: "OK PEACOCK TIME!"